libs/expo/betterangels/src/lib/screens/NotesHmis/HmisProgramNoteForm/HmisProvidedServices/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { ServiceRequestTypeEnum } from '../../../../apollo';
+import { HmisRequestedProvidedServices } from '../../../../ui-components';
+import { ViewHmisNoteQuery } from '../../HmisProgramNoteView/__generated__/HmisProgramNoteView.generated';
+
+interface IHmisProvidedServicesProps {
+  services?: ViewHmisNoteQuery['hmisNote']['providedServices'];
+}
+
+export default function ProvidedServices(props: IHmisProvidedServicesProps) {
```

Comment:
can we call this `HmisProvidedServices`? Unless leaving this name was intentional/serves a function